db/migrations/9.4.04.sql

```diff
@@ -0,0 +1,247 @@
+DO $$
+BEGIN
+   IF EXISTS (
+      SELECT 1
+      FROM information_schema.columns 
+      WHERE table_schema = 'public'
+            AND table_name = 'accounts'
+            AND column_name = 'identity_id'
+      ) 
+   THEN 
+      IF EXISTS (
+         SELECT 1
+         FROM information_schema.columns 
+         WHERE table_schema = 'public'
+               AND table_name = 'accounts'
+               AND column_name = 'identity_id'
+               AND is_nullable = 'NO' )
+      THEN
+         ALTER TABLE public.accounts ALTER COLUMN identity_id DROP NOT NULL;
+      END IF;
+   END IF;
+END $$;
```

Comment:
If you directly add in the alter query does that fail in some case? I think altering column to drop not null would not throw an error if the column was already nullable in the first place. Worth double checking